output/slide-deck.qmd

```diff
@@ -680,16 +611,11 @@ Mahugh](https://www.dougmahugh.com/envisioning-github/)
 ::: notes
 Use the "Post office" metaphor to explain:
 
--   Stage: Pack products (changes) to send
-
--   Commit: Check packages (changes) in to the postal service
-
-(You can check in several "packages" and the post officer can pick them up
-altogether)
-
--   Push: Send packages to their destination (changes to the repository)
+-   Stage: Pack products (changes) to send (you can "pack" several files or
+    changes in a single package)
 
--   Pull: Receive your packages (fetch changes and merge them)
+-   Commit: Check packages (changes) in to the postal service (each "commit" is
+    a package with several changes)
 :::
 
 ## (Even more) git jargon (\*)
```

Comment:
As noted above, might be worth combining with more jargon slide

Reply:
Check out my previous comments above. The thing is that I want to make a clear point that versioining and synchronizing are different things, so that people are aware that by just "committing" they don't have their latest changes remotely on GitHub. That's important because if that's not clear, it can lead to surprises if e.g. they commit without pushing and then don't find the latest changes on another computer.

Maybe I'm trying to teach it from a "system perspective" instead of a "user-centric" one, so I might be doing it against the same principles I was defending early on and you are right... Does it really make more sense to you to understand it as a whole "commit-push" lifecycle? If so, maybe it's worth trying to rethink these two sections and collapse them into a single one, as you say.